generators/app/templates/package.json

```diff
@@ -42,14 +42,14 @@
     "babel-register": "6.18.0",
     "babelify": "7.3.0",
     "eslint": "3.15.0",
-    "eslint-config-payscale": "0.0.5",
+    "eslint-config-payscale": "1.0.3",
     "eslint-plugin-react": "6.9.0",
     "eslint-teamcity": "1.3.1",
     "extract-text-webpack-plugin": "2.1.0",
     "husky": "0.14.3",
     "jsdom": "9.9.1",
     "lint-staged": "7.2.0",
-    "prettier": "1.14.0",
+    "prettier": "1.12.1",
```

Comment:
Just double checking, using the older version of prettier is intentional, right?

Reply:
Yeah that was on purpose. Wanted the version to match all our other projects with this standard setup.

Reply:
👍